charts/turbinia/Chart.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
```

Comment:
Can you add this (or any of the lock files or things that have hard-coded versions from this PR) to our our release documentation as something that needs to get updated sometimes?